modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -50,14 +50,17 @@ You can enclose a range of characters in square brackets, to match against all o
 | Matches any alphanumeric character.
 
 | `^`
-| Matches beginning of input.
-If the multi-line flag is set to `true`, also matches immediately after a line-break character.
-For example, `/^A/` does not match the '[.code]``A``' in "[.code]``an A``", but does match the '[.code]``A``' in "[.code]``An E``".
+| Matches the beginning of input.
+For example, `/^a/` matches `abb` and `ab`; but does _not_ match `ba`, `bab`, or `bba`.
+If the multi-line flag is set to `true`, `^` matches the start of _any line_ within the input string, instead of the beginning of the entire string.
 
-The '[.code]``^``' has a different meaning when it appears as the first character in a character set pattern.
-See https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions#special-negated-character-set[complemented character sets^] for details and an example.
+Used within square brackets, as the first character, `^` implies _negation_: therefore, `[^0-9]` matches against any character that is not a digit.
+
+| `$`
+| Matches the end of input.
+For example, `/$a/` matches `ba` and `bba`; but does _not_ match `abb`, `bab`, or `ab`.
```

Comment:
I haven't explicitly tested, but the general pattern is that the `$` represents the end of the line itself, rather than being a marker for "this is a string at the end of the line". E.g., I think this should be:

```
For example, `a$` matches `ba` and `bba`; but does _not_ match `abb`, `bab`, or `ab`.
```

See above for slashes around regexes.

Reply:
Thanks. I have changed accordingly.